### PR TITLE
Final approval

### PR DESCRIPTION
**Pull request by PowerLevel9000**
# Stock Market Analysis Web App

- Home Page
  - [ ] Navigation Bar with Home written with respective icons
  - [ ] Carousel of stock market images
  - [ ] Select your interest in the stocks with three options
    - [ ] Active Company (Default)
    - [ ] Top Gainers (Bullish)
    - [ ] Top Losers (Bearish)
  - [ ] company components with a respective bullish or bearish indicator of the symbol (green up arrow or down red arrow) or image
  - [ ] components with price and price change status 
  - [ ] components with a guided link with a right-facing arrow for the short details page of that company

- Short Details Page 
  - [ ] short Details About the company in the banner 
    - name 
    - price
    - price change
    - volume
    - market capital 
    - guide link for company website 
    - guided link for specific company Full details page and balance sheet page 
    - etc
  - [ ] on the bottom of the banner you will find different companies' performance strips with
    - symbol 
    - name 
    - price
    - indicator
    - and guided data fetch with right facing arrow for short details of the company
- Full Details Page
  - [ ] in this page you will see a banner for short details of the company (‼️‼️‼️sometime company provide a broken logo ‼️‼️‼️)
  - [ ] and a table with values and parameters that will tell you as many details as we have 
- Balance Sheet page 
  - [ ] a short Details of the company 
  - [ ]it is under construction formatted balance sheet (for the 5 years);
  

  # ‼️‼️‼️Problems ‼️‼️‼️

## My API has limited Call Request so try to send as few requests as possible by the way I provided the placeholder data if it breaks 
also, you can check the documentation of the API  <a href="https://site.financialmodelingprep.com/developer/docs/">API DOCUMENTATION</a>
and follow these steps for your API key  <a href="https://scribehow.com/shared/How_to_Get_an_API_Key_from_Financial_Modeling_Prep__n8QVaJJiR6KtjoOpPIkBbg">Follow steps </a>
  
  ## Happy coding
CatChangingTheClockChangingTheTimeGIF

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,221 @@
...
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/PowerLevel9000/stock-market/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
```
Comment (gemmen29) [inline README.md:205]: - [ ] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.).

File: README.md [hunk 2]
```diff
@@ -1,46 +1,221 @@
...
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> This link will guide you to my project
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/8d9134ee-50f3-49b0-91fd-5f3f68399a3d/deploy-status)](https://stock-analysis-io.netlify.app/)
+
+presentation video ![here](https://www.loom.com/share/c9ce2a2e0f1f4c7a80420b51f00f4ae3)
```
Comment (gemmen29) [inline README.md]: - [ ] Kindly fix the video when I press that link I wasn't able to view your video. I see the preview of the video, but I wasn't able to play the video. There is a white line in the middle of the play and the play button didn't appear.
image

File: src/index.css [hunk 3]
```diff
@@ -1,13 +1,10 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+:root {
```
Comment (gemmen29) [inline src/index.css:1]: - [ ] Kindly follow the same Typography as the original design as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.). The `font face` is different than the original design. You use the default `font family` of the browser `Times New Romans`. These are the fonts provided in the design guidelines.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @PowerLevel9000, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good UI ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @PowerLevel9000, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Functionalities working as expected :heavy_check_mark: 
- Nice presentation :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PowerLevel9000 — COMMENTED** [review]
@amejid @gemmen29 


Thanks everyone for your reviews 😉😉

**Comment by PowerLevel9000** [comment]
@amejid @gemmen29  hi respective CRs

I hope you are doing well, I am here to as you about optional suggestions for my project as I am having a week's break I decided to make my projects more attractive 

thanks in advance

**Pull request merged**